### PR TITLE
Add recipe for DBModelR version 0.1.0

### PR DESCRIPTION
**Pull request by LainPavot**
Hello!
I would like to add a recipe for an new R package: DBModelR.
This package in an ORM written in R, and I need it in one of my projects, deployed using conda dependencies.
This is the first recipe I made, so I hope I did nothing wrong.
Thanks! 

- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-dbmodelr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/LainPavot/DBModelR/archive/v{{ version }}.zip
+  md5: 27e1aa703c667f5d39ddcd38e1e728e7
+  sha256: 3126193656e11491ceaa5fe56d7f3c988c95870cba987b539d5fad2cc675317a
+
+build:
+  number: 0
+  script: Rscript ./test_install_package.R
```
Comment (dbast) [inline recipes/r-dbmodelr/meta.yaml:14]:
```suggestion
  script: R CMD INSTALL --build .
```

Comment (LainPavot) [reply recipes/r-dbmodelr/meta.yaml]: Thanks!
I've just commited your code enhancement suggestion.

**Comment by LainPavot** [comment]
Hello!
Is there any other errors in the repo..?

**Comment by LainPavot** [comment]
Hello!
Is there any other errors in the repo..?

@conda-forge/staged-recipes

File: recipes/r-dbmodelr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+package:
+  name: r-dbmodelr
+  version: {{ version }}
+
+source:
+  url: https://github.com/LainPavot/DBModelR/archive/v{{ version }}.zip
+  md5: 27e1aa703c667f5d39ddcd38e1e728e7
+  sha256: 3126193656e11491ceaa5fe56d7f3c988c95870cba987b539d5fad2cc675317a
+
+build:
```
Comment (bgruening) [inline recipes/r-dbmodelr/meta.yaml:12]: I think that should look like:

```
build:
  noarch: generic
  merge_build_host: true  # [win]
  number: 0
  rpaths:
    - lib/R/lib/
    - lib/
```

File: recipes/r-dbmodelr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: r-dbmodelr
+  version: {{ version }}
```
Comment (bgruening) [inline recipes/r-dbmodelr/meta.yaml:5]: Can you make it like that:

`  version: {{ version|replace("-", "_") }}`

**Review by bgruening — COMMENTED** [review]
Thanks @LainPavot! 

A few small comments to make it look like other recipes: https://github.com/conda-forge/r-dbplyr-feedstock/blob/063a2fb19fba414455664dd48797fd49d5820287/recipe/meta.yaml

I think this package can be noarch.

File: recipes/r-dbmodelr/meta.yaml (hunk 2 above)
Comment (LainPavot) [reply recipes/r-dbmodelr/meta.yaml:12]: Thank you for your advice!

**Comment by fgiacomoni** [comment]
@bgruening, Lain proposed asked fixes and CI just finish. Thank again to @dbast and you for your help.

**Review by bgruening — APPROVED** [review]

**Pull request merged**